FlagsmithClient/src/test/java/com/flagsmith/TraitsTests.kt

```diff
@@ -157,4 +183,20 @@ class TraitsTests {
             )
         }
     }
-}
\ No newline at end of file
+
+    @Test
+    fun testGetTransientIdentity() {
+        mockServer.mockResponseFor(MockEndpoint.GET_TRANSIENT_IDENTITIES)
+        runBlocking {
+            val result = flagsmith.getIdentitySync("transient-identity")
+            assertTrue(result.isSuccess)
+            assertTrue(result.getOrThrow().traits.isNotEmpty())
+            assertTrue(result.getOrThrow().flags.isNotEmpty())
+            assertEquals(
+                "electric pink",
+                result.getOrThrow().traits.find { trait -> trait.key == "favourite-colour" }?.stringValue
+            )
+            assertTrue(result.getOrThrow().traits.find { trait -> trait.transient == true }?.transient == true)
```

Comment:
What is this? 2 things: 

 1. I have no idea how this is even supposed to work since we're not setting any transient traits as part of this request. 
 2. This test is literally just searching for a trait that has `transient = true`... then asserting that `transient = true`

Reply:
Ah, I think (1) can be answer by the use of `MockEndpoint.GET_TRANSIENT_IDENTITIES`, but (2) still stands.